### PR TITLE
new changes in header and in footer

### PR DESCRIPTION
**Pull request by nishat-jahan-lisa**
header, footer changed with our logo

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -91,26 +92,37 @@ function MobileNavigation() {
...
             <div className="hidden md:block">
-              <NavLink href="/login">Sign in</NavLink>
+              <NavLink href="/login" className="text-black text-xl font-medium">
+                Sign in
+              </NavLink>
             </div>
             <Button href="/register" color="blue">
               <span>
-                Get started <span className="hidden lg:inline">today</span>
+                Get started <span className="hidden lg:inline"></span>
```
Comment (QAsRevenge) [inline src/components/Header.jsx:125]: Span no longer needed since its empy ;)

File: src/components/Footer.jsx
Comment (QAsRevenge) [inline src/components/Footer.jsx:1]: Lots of confusion about what the project is actually about but were not creating a new NetOnNet site so we shouldnt be using their name on our website. To my understanding were creating an internal 'dashboard' for us as a DevOps team, called Noffnet, following the practices of NetOnNet. We can talk more about this and really get going next week :)!.

File: src/images/logo.jpg
Comment (QAsRevenge) [inline src/images/logo.jpg:1]: Better to use SVGs for more consistent resolution across screens. A quick trick is to import the jpg into Figma and from there export it as a SVG!

**Review by QAsRevenge — CHANGES_REQUESTED** [review]
We can talk more about what the project is actually about in next week or see if we can coordinate a call with all of us this weekend. Till then and everybody is onboard lets hold off on PRs :).

File: src/images/logo.jpg
Comment (nishat-jahan-lisa) [reply src/images/logo.jpg:1]: ok but we will use the logo which you made, and sure we will export it as SVG

**Pull request closed without merging**